README.md

```diff
@@ -71,22 +67,31 @@ $ cargo install grpcio-compiler
 $ protoc --rust_out=. --grpc_out=. --plugin=protoc-gen-grpc=`which grpc_rust_plugin` example.proto
 ```
 
+
+### Option 2 - Programmatic Generation
+
+Programmatic generation can be used to generate Rust modules from proto files
+via your `build.rs` by using [protoc-grpcio](https://crates.io/crates/protoc-grpcio).
+
+For more information and examples see the
+[README](https://github.com/mtp401/protoc-grpcio/blob/master/README.md).
+
+
 To include this project as a dependency:
 
 ```
 [dependencies]
 grpcio = "0.2"
 ```
 
-Option 2 - Programmatic Generation
-----------------------------------
+### Feature `secure`
 
-Programmatic generation can be used to generate Rust modules from proto files
-via your `build.rs` by using [protoc-grpcio](https://crates.io/crates/protoc-grpcio).
+`secure` feature enables support for TLS encryption and some authentication machenism. In case you don't need it like working in intranet, you can also disable it:
```

Comment:
`secure` feature enables support for TLS encryption and some authentication **mechanism**.  When you do not need it, for example when working in intranet, you can disable it using the following command:

Please note that we usually use ''do not'', ''is not'' and ''will not" instead of "don't", "isn't" and "won't" in technical documents.